admin/views/pupil-pin/generated-pins-list.ejs

```diff
@@ -9,6 +9,16 @@
 
 <div class="grid-row">
     <div class="column-two-thirds">
+        <% if (error) { %>
+        <div class="error-summary" role="group" aria-labelledby="error-summary-heading-1">
+            <h2 class="heading-medium error-summary-heading" id="error-summary-heading-1">
+                <%= error %>
+            </h2>
+            <p>Multiplication tables check helpline<br>
+                0345 278 8080
```

Comment:
I suspect this helpline number is used in multiple places - it should be in config though.
@GuyHarwood, @pris54  - thoughts?

Reply:
@jon-shipley added in config